AmilcarRosadoPat/src/main/java/com/applause/auto/pageframework/pages/HomePageAurora.java

```diff
@@ -0,0 +1,73 @@
+package com.applause.auto.pageframework.pages;
+
+import java.lang.invoke.MethodHandles;
+
+import com.applause.auto.framework.pageframework.util.logger.LogController;
+import com.applause.auto.framework.pageframework.web.AbstractPage;
+import com.applause.auto.framework.pageframework.web.WebElementLocator;
+import com.applause.auto.framework.pageframework.web.factory.WebDesktopImplementation;
+import com.applause.auto.framework.pageframework.web.factory.WebPhoneImplementation;
+import com.applause.auto.framework.pageframework.web.factory.WebTabletImplementation;
+import com.applause.auto.framework.pageframework.webcontrols.Dropdown;
+import com.applause.auto.framework.pageframework.webcontrols.EditField;
+
+@WebDesktopImplementation(HomePageAurora.class)
+@WebTabletImplementation(HomePageAurora.class)
+@WebPhoneImplementation(HomePageAurora.class)
+public class HomePageAurora extends AbstractPage {
+
+	protected final static LogController LOGGER = new LogController(MethodHandles.lookup().getClass());
+
+	@Override
+	protected void waitUntilVisible() {
+
+		// If we query for this element immediately the page doesn't load so we
+		// need a short wait
+		// syncHelper.suspend(1000);
+		syncHelper.waitForElementToAppear(getDestinoTextBox());
+	}
+
+	/*
+	 * Public Actions
+	 */
+	public void Destination(String Destino) {
+		LOGGER.info(String.format("Entering destination %s into search field", Destino));
+		getDestinoTextBox().setText(Destino);
+	}
+
+	public boolean ListHotels() {
+		boolean Lista = true;
+
+		/*
+		 * List<String> ListaAjax = new ArrayList<String>();
+		 * 
+		 * for (int f = 0; f <= 10; f++) { ListaAjax.add(getListAjax().getText());
+		 * 
+		 * }
+		 * 
+		 * System.out.println(ListaAjax);
+		 */
+		syncHelper.waitForElementToAppear(getListAjax());
+		System.out.println(getListAjax().getWebElement().getText());
+		if (getListAjax().getText().contains("No se encontraron resultados")) {
+			Lista = false;
+		}
+		return Lista;
```

Comment:
como funciona listHotels?

Reply:
Ahi lo unico que espera de mi metodo es que me retorne un boolean, para ello inicialice una variable tipo boleano por default con true, tengo un metodo que se llama getListAjax que es en donde se encuentra la cajita que me devuelve los resultados entonces ahi le puse un waitforelement que hace que espere que sea visible el elemento parra poder continuar, ahi solo le puse un print para ver que me   devuleva mi lista de resultados ya con el gettext y ahi le puse una condicion en donde si getListAjax contiene mi texto me ponga un false ya que no me devolvia resultados y si en caso no contiene eso que siga siendo true, y lo que esta arriba asi lo queria hacer al principio pero siempre me agrega el primer valor de getListAjax  las veces que le indico en mi for. y para validar en de ListHotels le puse en el test principal un assert en donde si es false se termine de ejecutar y si es true que continue

```suggestion
		return Lista;
```